content/en/docs/concepts/sdk-configuration/general-sdk-configuration.md

```diff
@@ -0,0 +1,129 @@
+---
+title: "General SDK Configuration"
+description: >-
+ General-purpose environment variables for configuring an OpenTelemetry SDK.
+weight: 1
+---
+
+The following environment variables are respected by every OpenTelemetry SDK,
+regardless of language.
+
+## `OTEL_SERVICE_NAME`
+
+Sets the value of the
+[`service.name`](/docs/reference/specification/resource/semantic_conventions/#service) resource
+attribute.
+
+**Default value:** `"unknown_service"`
+
+If `service.name` is also provided in `OTEL_RESOURCE_ATTRIBUTES`, then
+`OTEL_SERVICE_NAME` takes precedence.
+
+**Example:**
+
+`export OTEL_SERVICE_NAME="your-service-name"`
+
+## `OTEL_RESOURCE_ATTRIBUTES`
+
+Key-value pairs to be used as resource attributes. See [Resource
+SDK](/docs/reference/specification/resource/sdk#specifying-resource-information-via-an-environment-variable)
+for more details.
+
+**Default value:** Empty.
+
+See [Resource semantic
+conventions](/docs/reference/specification/resource/semantic_conventions/#semantic-attributes-with-sdk-provided-default-value)
+for semantic conventions to follow for common resource types.
+
+**Example:**
+
+`export OTEL_RESOURCE_ATTRIBUTES="key1=value1,key2=value2"`
+
+## `OTEL_TRACES_SAMPLER`
+
+Specifies the Sampler to be used by the SDK. Combining samples is done via an
```

Comment:
I don't understand the 'combining samples' part here -- you can only specify one sampler, correct?

Reply:
Technically no (e.g., can specify both parent-based and traceidratio-based). But then combining things like alwayson and alwaysoff is nonsense. 

Guess I should word it something to the effect of "[these combinations are valid](https://github.com/open-telemetry/opentelemetry-go/blob/1f5b159161e0ff54df4677284fa7d6718dc9186d/sdk/trace/sampler_env.go#L32-L34)"

Reply:
Alrighty this should be addressed now